### ISSUE TITLE
Re: 'Deploy an emergency banner'

### ISSUE THREAD

**Issue by MalcolmVonMoJ** [alphagov/govuk_publishing_components#3949]
Problem with 'Deploy an emergency banner' (https://docs.publishing.service.gov.uk/manual/emergency-publishing.html)

The Local Emergency Banner doesn't seem to pass colour contrast checks.  I'm getting [an AA fail on WebAIM Colour contrast check](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=29A298).

**Comment by andysellick** [comment]
Thanks for raising this @MalcolmVonMoJ. I believe we're talking about the turquoise colour for [Localised large scale emergency](https://docs.publishing.service.gov.uk/manual/emergency-publishing.html#localised-large-scale-emergency-level-2-or-category-1)? You're right, the colour there (`#28a197`) fails the colour contrast test, but I think the screenshot is out of date. The [current version of the component](https://components.publishing.service.gov.uk/component-guide/emergency_banner/local_emergency) uses `#00847d`, which passes everything apart from the AAA test for normal text. There's obviously room for improvement there, so I'll leave this issue open for now while we see what better colour could be used.
